src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/PackTask.cs

```diff
@@ -75,6 +77,14 @@ public IPackTaskLogic PackTaskLogic
 
         public override bool Execute()
         {
+#if DEBUG
```

Comment:
This doesn't seem like a good default for debug builds. Most people don't care about attaching the debugger to this bit of code.

Also, it's useful to have a `Console.WriteLine` which displays the PID.

I think this should be commented out by default.